### PR TITLE
Ready for the relase of v2.0

### PR DESCRIPTION
**Pull request by FoxWorn3365**
- Added new items
- Fixed some bugs

### CODE REVIEW

File: UncomplicatedCustomItems/API/Features/SummonedCustomItem.cs [hunk 1]
```diff
@@ -133,12 +128,40 @@ internal void SetProperties()
...
+                        break;
+
+                    case CustomItemType.FlashGrenade:
+                        ((FlashGrenade)Item).PinPullTime = ((IFlashGrenadeData)CustomItem.CustomData).PinPullTime;
+                        ((FlashGrenade)Item).Repickable = ((IFlashGrenadeData)CustomItem.CustomData).Repickable;
+                        ((FlashGrenade)Item).MinimalDurationEffect = ((IFlashGrenadeData)CustomItem.CustomData).MinimalDurationEffect;
+                        ((FlashGrenade)Item).AdditionalBlindedEffect = ((IFlashGrenadeData)CustomItem.CustomData).AdditionalBlindedEffect;
+                        ((FlashGrenade)Item).SurfaceDistanceIntensifier = ((IFlashGrenadeData)CustomItem.CustomData).SurfaceDistanceIntensifier;
+                        ((FlashGrenade)Item).FuseTime = ((IFlashGrenadeData)CustomItem.CustomData).FuseTime;
+                        break;
```
Comment (SpGerg) [inline UncomplicatedCustomItems/API/Features/SummonedCustomItem.cs]: HOLY SHIT, can you cast only one time? It is SERIOUSLY shit

File: UncomplicatedCustomItems/API/Utilities.cs [hunk 2]
```diff
@@ -269,5 +341,17 @@ internal static void SummonCustomItem(ICustomItem CustomItem)
                 }
             }
         }
+
+        internal static IEnumerator<float> PainkillersCoroutine(Player player, IPainkillersData Data)
+        {
+            float TotalHealed = 0;
```
Comment (SpGerg) [inline UncomplicatedCustomItems/API/Utilities.cs:327]: totalHealed

**Review by SpGerg — CHANGES_REQUESTED** [review]
Also, add all variants of custom items in config (weapon, grenade e.t.c)

File: UncomplicatedCustomItems/API/Utilities.cs (hunk 2 above)
Comment (FoxWorn3365) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: only args-given vars are camelCase, vars are PascalCase

File: UncomplicatedCustomItems/API/Features/SummonedCustomItem.cs (hunk 1 above)
Comment (FoxWorn3365) [reply UncomplicatedCustomItems/API/Features/SummonedCustomItem.cs]: ok daddy

File: UncomplicatedCustomItems/API/Utilities.cs (hunk 2 above)
Comment (SpGerg) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: send me an article where this is written

Comment (FoxWorn3365) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: lots of people uses it

Comment (SpGerg) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: Exiled developers not, plugin developers on nwapi and exiled not, server where i making plugins not too.

Comment (FoxWorn3365) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: this still is not a problem tho

Comment (SpGerg) [reply UncomplicatedCustomItems/API/Utilities.cs:327]: just change.

**Pull request closed without merging**